Library/Homebrew/formula.rb

```diff
@@ -127,6 +127,11 @@ class Formula
   # @private
   attr_accessor :local_bottle_path
 
+  # When performing a build, test, or other loggable action, indicates which
+  # log file location to use.
```

Comment:
I'm definitely against overengineering this and prematurely constraining this to a fixed set of symbols. Feel free to leave everything as-is; and thanks for your thoughts!